app/Helpers/EchoHelper.php

```
@@ -70,9 +70,8 @@ public static function AutoLink($str, $type = 'both', $popup = false): void
      * @originalFile echo_helper.php
      */
     public static function CsrfField(): void
-    {
-        $CI = & get_instance();
-        echo '<input type="hidden" name="' . $CI->config->item('csrf_token_name');
+    { // TODO: Replace with Laravel patterns
+        echo '<input type="hidden" name="' . config('csrf_token_name');
         echo '" value="' . $CI->security->get_csrf_hash() . '">';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$CI'. (undefined)

(UnusedLocalVariable)